### ISSUE TITLE
Wrong IE11 implementation of Date.toLocaleString

### ISSUE THREAD

**Issue by andresmrm** [Financial-Times/polyfill-service#497]
Hello! Not sure if this is the right place for this, but if I do:

    (new Date).toLocaleString('en', { month: '2-digit', day: '2-digit' })

in a Firefox or Chromium (without polyfill) or even in IE10 after polyfilled by polyfill-service, I get:

    "10/18"

But in IE11 I am getting:

    "10 18"

(in my native locale (pt-BR) the result is even more different (other browsers: `"18/10"`, IE11: `"18 de 10"`))

I think that IE11 says it implements Intl, and that's why polyfill-service is not sending the polyfill for it. But I thought it woul be better to report... Maybe you would pefer to force the polyfill in this case. (I think I'll force it using the querystring for polyfill-service)

(I tested IE11 using BrowserStack, not sure if a "local" installed IE11 would behave different, but should not...)

**Comment by triblondon** [comment]
Yes, I suspect `Intl` is read only in IE11 so we can't polyfill it.

**Comment by triblondon** [comment]
OK, looks like `Date.prototype.toLocaleString` is assignable in IE11 so this could potentially be fixed.

**Comment by triblondon** [comment]
@caridy Is this something to fix in Intl?

**Comment by caridy** [comment]
We could fix it in Intl.js, but not before we split it into pieces, which is an ongoing effort at the moment with no ETA just yet. I will follow up with MS folks this week as well.

**Comment by caridy** [comment]
I wonder what's going on with the built-in section in kangax table (http://kangax.github.io/compat-table/es6/#closure). I see WeakMap, WeakSet, Promise, Proxy, Reflect, and most symbols not implemented after using `--rewrite_polyfills`. Follow up: https://github.com/kangax/compat-table/issues/747

**Comment by triblondon** [comment]
OK, will park this issue for the moment then.

**Comment by caridy** [comment]
Backpointer to MS Bug report: https://connect.microsoft.com/IE/feedbackdetail/view/2535943/ie11-date-prototype-tolocalestring-produces-incorrect-localized-date

**Comment by zappan** [comment]
@caridy @triblondon MS resolved the above issue as non-reproducible.
I'm in a search for a `Number.toLocaleString` polyfill, wish to get a feedback if you will push with the implementation of this one. Thx

**Comment by caridy** [comment]
@zappan if you have a link to the ticket that they have marked as non-reproducible, I can dig in, I will be a MS in about a week for TC39, and I can discuss it with them directly.

**Comment by zappan** [comment]
@caridy It's the above link you posted
https://connect.microsoft.com/IE/feedbackdetail/view/2535943/ie11-date-prototype-tolocalestring-produces-incorrect-localized-date

**Comment by JakeChampion** [comment]
Hi @caridy, have you been able to get in contact with MS about this issue? Is this issue still waiting upon the task of breaking Intl down into smaller pieces?

**Comment by caridy** [comment]
@JakeChampion here is the issue to break Intl into pieces: https://github.com/andyearnshaw/Intl.js/issues/242

**Comment by triblondon** [comment]
This is still awaiting progress on https://github.com/andyearnshaw/Intl.js/issues/242
